### ISSUE TITLE
Automatically setup velocity modern forwarding

### ISSUE THREAD

**Issue by gamer50082** [AkramLZ/ServerSync#1]
Velocity has modern forwarding and it required it proxy modern forwarding is enabled could you make the plugin automatically setup on server side

**Comment by AkramLZ** [comment]
Hello!
Modern forwarding shouldn't be an issue since all paper servers share the same secret key, which means you don't need to configure velocity each time you add a new paper server.

**Comment by gamer50082** [comment]
no u need to manually add the secret and enable it

**Comment by AkramLZ** [comment]
> no u need to manually add the secret and enable it

Yes, you need to add it in the **Paper** server, not in the Velocity proxy, which means you are not going to modify or reload the proxy server (and that's the point of the plugin).

## How to configure modern forwarding for Velocity?
1. Copy the secret key from your velocity server, it will be in a file called `forwarding.secret`
2. Go to your paper server and open the file `config/paper-global.yml`
3. Set `proxies.velocity.enabled` to `true`
4. Paste your key in the section `proxies.velocity.secret` to match your velocity secret forwarding key.
For more information check [Velocity documentation](https://docs.papermc.io/velocity/player-information-forwarding#configuring-modern-forwarding).

> If you mean anything else please explain, thanks!

**Comment by gamer50082** [comment]
i was talking about the plugin doing all that automatically

**Comment by AkramLZ** [comment]
> i was talking about the plugin doing all that automatically

I think it is a good idea to implement, thank you!
It is going to be implemented in the next update.
